meta-lmp-base/recipes-containers/docker-compose/docker-compose_2.4.1.bb

```diff
@@ -20,6 +26,9 @@ GO_EXTRA_LDFLAGS = "-w -X ${GO_IMPORT}/internal.Version=${PV}"
 go_do_compile() {
 	export TMPDIR="${GOTMPDIR}"
 	mkdir -p ${B}/cli-plugins/bin
+	${GO} mod download -modcacherw
+	chmod -R +w ${B}/pkg/mod
```

Comment:
Do we need chmod with modcacherw?

Reply:
```
-       chmod -R +w ${B}/pkg/mod
-       cp ${S}/src/github.com/docker/cli/cli/config/config.go ${B}/pkg/mod/github.com/docker/cli@v20.10.3-0.20220309205733-2b52f62e9627+incompatible/cli/config/config.go
+       cp -f ${S}/src/github.com/docker/cli/cli/config/config.go ${B}/pkg/mod/github.com/docker/cli@v20.10.3-0.20220309205733-2b52f62e9627+incompatible/cli/config/config.go
```

Seems to work ^^

Reply:
I guess `cp` will fail without `modcacherw`

Reply:
I tested with cp -f and it worked fine locally.

Reply:
Do you mean without `-modcacherw`?

Reply:
i was thinking it was needed for cleanup logic (rm was failing), but i might have just copied it from some other recipe without thinking it through.